Macros/ACS_summary_geo.sas

```diff
@@ -1086,6 +1086,20 @@
       mNumbroadband_&_years. mNumbroadbandonly_&_years. mNumsatellite_&_years. mNumsatelliteonly_&_years. mNumotheronly_&_years. 
       mNumaccesswosub_&_years. mNumnointernet_&_years. mNum1orMoreComp_&_years. mNumhhdefined_&_years. mNumwithinternet_&_years.
     ;
+	/*Rodrigo added Homeownership and Renting Indicators*/
+	%let moe2017plus = 
+		Mort_CB_&_years. Mort_SCB_&_years. Mort_CBC_&_years. NoMort_CB_&_years. NoMort_SCB_&_years. NoMort_CBC_&_years. AllOwn_CB_&_years. AllOwn_SCB_&_years. AllOwn_CBC_&_years. Rent_CB_&_years. Rent_SCB_&_years. Rent_CBC_&_years. 
+		NHW_Mort_CB_&_years. NHW_Mort_SCB_&_years. NHW_Mort_CBC_&_years. NHW_NoMort_CB_&_years. NHW_NoMort_SCB_&_years. NHW_NoMort_CBC_&_years. NHW_AllOwn_CB_&_years. NHW_AllOwn_SCB_&_years. NHW_AllOwn_CBC_&_years. NHW_Rent_CB_&_years. NHW_Rent_SCB_&_years. NHW_Rent_CBC_&_years. 
+		H_Mort_CB_&_years. H_Mort_SCB_&_years. H_Mort_CBC_&_years. H_NoMort_CB_&_years. H_NoMort_SCB_&_years. H_NoMort_CBC_&_years. H_AllOwn_CB_&_years. H_AllOwn_SCB_&_years. H_AllOwn_CBC_&_years. H_Rent_CB_&_years. H_Rent_SCB_&_years. H_Rent_CBC_&_years. 
+		B_Mort_CB_&_years. B_Mort_SCB_&_years. B_Mort_CBC_&_years. B_NoMort_CB_&_years. B_NoMort_SCB_&_years. B_NoMort_CBC_&_years. B_AllOwn_CB_&_years. B_AllOwn_SCB_&_years. B_AllOwn_CBC_&_years. B_Rent_CB_&_years. B_Rent_SCB_&_years. B_Rent_CBC_&_years. 
+		API_Mort_CB_&_years. API_Mort_SCB_&_years. API_Mort_CBC_&_years. API_NoMort_CB_&_years. API_NoMort_SCB_&_years. API_NoMort_CBC_&_years. API_AllOwn_CB_&_years. API_AllOwn_SCB_&_years. API_AllOwn_CBC_&_years. API_Rent_CB_&_years. API_Rent_SCB_&_years. API_Rent_CBC_&_years. 
+		IOM_Mort_CB_&_years. IOM_Mort_SCB_&_years. IOM_Mort_CBC_&_years. IOM_NoMort_CB_&_years. IOM_NoMort_SCB_&_years. IOM_NoMort_CBC_&_years. IOM_AllOwn_CB_&_years. IOM_AllOwn_SCB_&_years. IOM_AllOwn_CBC_&_years. IOM_Rent_CB_&_years. IOM_Rent_SCB_&_years. IOM_Rent_CBC_&_years. 
+		APIIOM_Mort_CB_&_years. APIIOM_Mort_SCB_&_years. APIIOM_Mort_CBC_&_years. APIIOM_NoMort_CB_&_years. APIIOM_NoMort_SCB_&_years. APIIOM_NoMort_CBC_&_years. APIIOM_AllOwn_CB_&_years. APIIOM_AllOwn_SCB_&_years. APIIOM_AllOwn_CBC_&_years. APIIOM_Rent_CB_&_years. APIIOM_Rent_SCB_&_years. APIIOM_Rent_CBC_&_years.
+
+	%else %if &_last_year. >=2017 %then %do;    
```

Comment:
@RAGarMir There was another %else %if that you inserted without a corresponding %if before it. This one should also be changed to a %if. A semicolon is also missing at the end of the previous statement. `APIIOM_Rent_CBC_&_years. ;` These generated the following errors.

```
ERROR: There is no matching %IF statement for the %ELSE.
ERROR: A dummy macro will be compiled.
ERROR: Macro keyword IF appears as text.
ERROR: There is no matching %DO statement for the %END. This statement will be ignored.
```

I pushed these fixes, but will have to review the rest of the code later today.